packages/xstate-wallet/src/workflows/approve-budget-and-fund.ts

```diff
@@ -95,9 +113,22 @@ export const approveBudgetAndFundWorkflow = (
     updateBudget: (context: WorkflowContext, event) => {
       return store.updateOrCreateBudget(context.budget);
     },
-    createAndFundLedger: (context: WorkflowContext, event) => {
-      // TODO: Hook up to workflow when it exists
-      return Promise.resolve();
+    createAndFundLedger: ({player, hub, budget}: WorkflowContext) => {
+      // TODO: Abstract into function that takes a budget and returns a outcome
+      const ethBudget = budget.budgets[ETH_ASSET_HOLDER_ADDRESS];
```

Comment:
`budget.budgets` makes it seem like one of `budget` or `budgets` is poorly named -- why does a budget have many budgets?. Maybe `budget.forToken[...]`?

Reply:
I've renamed to `forAsset` for now.